### PR TITLE
Create demo commitment blog for the demo

### PR DESCRIPTION
**Pull request by wpschaub**
- Update my bio
- Propose a post to show our character through the blog lens

### CODE REVIEW

File: content/we-make-a-difference.md [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+| CHARACTER | OUR COMMITMENT |
+|-----------|----------------|
+| ![Caring](/images/blog-caring.png) | We care about the technical community we collaborate with. |
+| ![Fair](/images/blog-fair.png) | We build trust and credibility by showcasing our cutting edge technical accomplishements.  |
+| ![Knowledgeable](/images/blog-knowlegeable.png) | We make a difference by clearly sharing technical events, challenges, and 2AM call reflections. |
+| ![Responsive](/images/blog-responsive.png) | We minimise the risk that bold innovation may introduce, by sharing what is relevant, when it is needed. |
+| ![Respectful](/images/blog-respectful.png) | We review all proposed content and its tone, to keep it accurate and respectful. |
+| ![Understandable](/images/blog-understandable.png) | We strive for clarity and simplicity when covering complex technical topics. |
+
+> Should you find any incorrect, out-of-date, upsetting, or offensive content you can contact @TBD@.
```
Comment (wpschaub) [inline content/we-make-a-difference.md:20]: We need an address to replace @TBD@ placeholder.

**Review by solicitfeedback — CHANGES_REQUESTED** [review]
Hi Willy, looks great, I'd like to ask if you could improve the last paragraph?

**Review by vegantiger8 — APPROVED** [review]
I'm Vegan but I love Cheese ;) - Thanks Willy

**Comment by wpschaub** [comment]
@vegantiger8 @solicitfeedback I will close (**abort**) this PR and park my changes of the "denmo" post. **We need marketing to first consider** whether we can post such a post. I will split the author update into a seperate PR and submit later today.

**Pull request closed without merging**